tests/func/test_breakpoints.py

```diff
@@ -398,3 +399,84 @@ def code_to_debug():
         output = session.all_occurrences_of(Event('output', ANY.dict_with({'category': 'stdout'})))
         output = sorted(int(o.body['output'].strip()) for o in output if len(o.body['output'].strip()) > 0)
         assert list(range(0, 10)) == output
+
+
+def test_invalid_breakpoints(pyfile, run_as, start_method):
+    @pyfile
+    def code_to_debug():
```

Comment:
@qubitron, can you take a look at this (just the body of `code_to_debug`) to see that it captures various scenarios correctly? The intent is expressed in comments - bpN-requested is where the user might set it, and bpN-expected is where it should end up.

If you can think of any other such cases, let me know, so that I can add them here.